lib/router/src/router.tsx

```diff
@@ -55,15 +56,15 @@ interface QueryLinkProps {
   children: React.ReactNode;
 }
 
-const base = document.location.pathname + '?';
```

Comment:
Why?

Reply:
Also can we move the `?` into the caller code? This looks kinda weird to me

Reply:
I think the function is there to be able to read the current value of document.location (because this changes between story selections)

Reply:
To evaluate this code lazily.

Otherwise loading this module in node throws.

Reply:
`document.location.pathname` shouldn't change between stories, I think.